src/System.Management.Automation/engine/remoting/commands/EnterPSHostProcessCommand.cs

```diff
@@ -707,7 +707,7 @@ internal static IReadOnlyCollection<PSHostProcessInfo> GetAppDomainNamesFromProc
                                     else if (process.ProcessName.Equals(pName, StringComparison.Ordinal))
```

Comment:
@PaulHigin, your last commit had 1 failures in `PowerShell-CI-windows`
ForEach-Object -Parallel Functional Tests.Verifies parallel task queuing
```powershell
Expected the actual value to be greater than 5, but got 5.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1: line 369
369:         $results[2] | Should -BeGreaterThan 5
```